src/handlers/fileResolver.js

```diff
@@ -18,6 +39,10 @@ const resolveSync = enhancedResolve.create.sync({
  */
 function resolve(relPath = "", currentFile = "") {
   const currentDir = path.dirname(currentFile);
+  if (!relPath.match(/^(\.|\/)/)) {
```

Comment:
Also about this `if`, I suggest moving it after line `53` inside catch phrase, in that way all the normal cases (1. relative imports, 2. node_modules imports) are handled first, then if it is not found there it can look for babel aliases